src/app/shared/entry.ts

```diff
@@ -315,6 +387,18 @@ export abstract class Entry implements OnInit, OnDestroy {
     return recentVersions;
   }
 
+  getSortedWorkflowVersions(versions: Array<WorkflowVersion>, defaultVersion: WorkflowVersion): Array<WorkflowVersion> {
+    let sortedWorkflowVersions: Array<WorkflowVersion> = [];
```

Comment:
Suggest making this one line and using `const` modifier instead of `let`. Generally, unless you want the variable to be modifiable, you should prefer const.